### ISSUE TITLE
suspicious interaction - Summer fruit tortrix moth (Adoxophyes orana) pathogen of itself

### ISSUE THREAD

**Issue by jhpoelen** [millerse/Ecological-Database-of-the-World-s-Insect-Pathogens#4]
From https://www.globalbioticinteractions.org/?interactionType=parasiteOf&sourceTaxon=Adoxophyes%20orana (see attached screenshot), it appears that Summer fruit tortrix moth (_Adoxophyes orana_) is a pathogen of itself. This strikes me as suspicious.

![screenshot from 2017-12-27 12-38-51](https://user-images.githubusercontent.com/1084872/34392815-69e77f16-eb03-11e7-893f-2efc5e644161.png)

**Comment by jhpoelen** [comment]
Root cause of this reported issue lies in suspicious name mappings in GloBI's Taxon Graph from "Adoxophyes orana NPV" -x-> "Adoxophyes orana" . The issue does *not* appear to originate from the data transcribed by @millerse . 

This issue appears to have been fixed in GloBI Taxon Graph v0.3.33 (four years later). Waiting for changes to propagate.

See also https://github.com/globalbioticinteractions/globalbioticinteractions/issues/672 .

Related data:

```
$ elton interactions --cache-dir /var/cache/elton/datasets "millerse/Ecological-Database-of-the-World-s-Insect-Pathogens" | grep "Adoxophyes orana NPV"
listing interactions [millerse/Ecological-Database-of-the-World-s-Insect-Pathogens]... https://en.wiktionary.org/wiki/support			EOL:370467	Adoxophyes orana											http://purl.obolibrary.org/obo/RO_0002557	hasPathogen						EOL:41592680	Adoxophyes orana NPV				GEONAMES:1861060	Japan			Martignoni, M.E., and Iwai, P.J., 1986. A Catalogue of Viral Diseases of insects, Mites, and Ticks. USDA Forest Service, Pacific Northwest Research Station, Portland, OR, General Technical Report PNW-195, 4th edition,51 pp.; Zimmermann, G. and Weiser, J. 1991. Pathogens and diseases. pp. 253-271. In van der Geest, L. P. S. and Evenhuis, H.H. [eds.], Tortricid Pests: Their Biology, Natural Enemies and Control. Elsevier Science Publishers, Amsterdam.	millerse/Ecological-Database-of-the-World-s-Insect-Pathogens	Sarah E Miller. 12/13/2016. Species associations manually extracted from Onstad, D.W. EDWIP: Ecological Database of the World's Insect Pathogens. Champaign, Illinois: Illinois Natural History Survey, [23/11/2016]. http://insectweb.inhs.uiuc.edu/Pathogens/EDWIP.	https://github.com/millerse/Ecological-Database-of-the-World-s-Insect-Pathogens/archive/187c16c8e664860d07e20f8a5ce209836e609c0b.zip	2021-06-19T05:09:54.027Z	e5caddef01085a38b1bece79e9cd268fb1f1a7b49e4165ef9ef5f05739f5fc8e	0.11.0
done.
jhpoelen@globi5:~/tmp/name-issue$ echo -e "EOL:41592680\t" | nomer append
using default matcher [globi-taxon-cache]
EOL:41592680		NONE		EOL:41592680							
jhpoelen@globi5:~/tmp/name-issue$ echo -e "EOL:41592680\tAdoxophyes orana NPV" | nomer append
using default matcher [globi-taxon-cache]
EOL:41592680	Adoxophyes orana NPV	NONE	Adoxophyes orana NPV	EOL:41592680							
```

**Comment by jhpoelen** [comment]
After propagation of GloBI taxon graph v0.3.33, the claim that Summer fruit tortrix moth (Adoxophyes orana) is a pathogen of itself no longer occurs. Instead, the actual reported interactions between pathogenic viruses and their hosts Summer fruit tortrix moth (Adoxophyes orana) are listed via https://www.globalbioticinteractions.org/?interactionType=parasiteOf&sourceTaxon=Adoxophyes%20orana . See attached screenshot.

@millerse Apologies for the 4 years delay in addressing this issue on my end.
